spec.md

```diff
@@ -2044,6 +2049,10 @@ To summarize, optional groups have the following rules:
    outside the group or in parent groups.
 1. All other statements and expressions are allowed inside a group instance.
 1. Group instances may have output ports of passive, probe type.
+1. A first module that instantiates a second module may access ports of the
+   second module declared in the second module's group instance within a group
+   instance in the first module if the group instances are the same or if the
+   second module's group instance is a parent of the first module's group.
```

Comment:
```suggestion
   first module's group instance is within the nested group hierarchy of the second module's group.
```

Reply:
or something -- now that we have two hierarchy systems going i found this confusing the way it was written

Reply:
This condition is difficult to write. Cleaning this up in `fixup!` commit.